site/admission/index.html

```diff
@@ -164,18 +194,20 @@ <h4>Social Media Icons</h4>
 <p>lorem ipsum</p>
 <h2>Upcoming Admissions Events</h2>
 <div>
-  <div>Nov 10</div>
-  <p>Sign up for open house</p>
-</div>
-<div>
+  <img src="../assets/images/placeholder-image-rectangle.png" alt="placeholder-image-rectangle" width="275"
```

Comment:
I think we spoke about this, but you can remove the height and width from all `img` tags